contracts/access/IAccessControl.sol

```diff
@@ -31,7 +31,7 @@ interface IAccessControl {
      * @dev Emitted when `account` is granted `role`.
      *
      * `sender` is the account that originated the contract call, an admin role
-     * bearer except when using {AccessControl-_setupRole}.
+     * bearer except when using {AccessControl-_grantRole}.
```

Comment:
Outdated reference.

Reply:
This was already fixed in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4924, though we didn't cherry pick it to the releaase branch. Let's merge this and then cherry pick to the release branch so the documentation gets updated accordingly.